app/src/NativeBase/Containers/ListContainer.tsx

```diff
@@ -7,23 +7,30 @@
 
 /* eslint-disable @typescript-eslint/no-shadow */
 
+import { Text, Heading, VStack } from 'native-base'
+
 import React, { useEffect, useState } from 'react'
-import { Text } from 'react-native'
 import { useDispatch, useSelector } from 'react-redux'
 import styled from 'styled-components/native'
-import { EventSearch } from './EventSearchContainer'
+import { EventSearch } from '../../Containers/EventSearchContainer'
```

Comment:
Small thing but the syntax we normally use to reference a different top-level directory inside `app/src` is '@/Containers/EventSearchContainer'